src/System.Text.Encodings.Web/src/project.lock.json

```diff
@@ -1,18 +1,15 @@
 {
-  "locked": true,
+  "locked": false,
```

Comment:
I think this should technically be locked, although you don't have any non-specific versions listed.